ksp_plugin/frames.hpp

```diff
@@ -37,15 +37,19 @@ using AliceWorld = Frame<serialization::Frame::PluginTag,
 
 // The barycentric reference frame of the solar system.
 // The basis is the basis of |AliceWorld| at |UniversalTimeEpoch|.
-// TODO(egg): it *should* be the barycentric frame. For the moment we're using
-// the velocity of the sun at the time of construction as our reference.
-// The origin is the position of the sun at the instant |initial_time| passed at
-// construction.
-// TODO(phl): this has been flipped, change the value of the protobuf enum and
```

Comment:
As far as I can tell the second TODO was precisely about `PRE_BOREL_BARYCENTRIC`, and was addressed before releasing Borel ([blame](https://github.com/mockingbirdnest/Principia/blame/5fcca7e/ksp_plugin/frames.hpp#L44), [commit](https://github.com/mockingbirdnest/Principia/commit/5c017d6f0b50394346119a1310e29a795d783839), note that it's in since Borel, [pull request containing the commit](https://github.com/mockingbirdnest/Principia/pull/495), note that it's the one that flips the universe).
The first one was incidentally addressed in #835, when switching to Jacobi coordinates for Keplerian elements.